docs/source/running-autometa.rst

```diff
@@ -2,85 +2,266 @@
+
+    # Archaeal markers
+    --orfs <path/to/amino_acid/ORFs.faa> --kingdom archaea --hmmscan <path/to/archaea.hmmscan.tsv> \
+    --out <path/to/archaea.markers.tsv>
+
+    # Bacterial markers
+    --orfs <path/to/amino_acid/ORFs.faa> --kingdom bacteria --hmmscan <path/to/bacteria.hmmscan.tsv> \
+    --out <path/to/bacteria.markers.tsv>
+
+The above command would generate the following files:
+
+- Table having the archaeal and bacterial marker genes identified on each contig called ``<path/to/archaea.markers.tsv>`` and ``<path/to/bacteria.markers.tsv>`` for archaea and bacteria respectively
+
+You can view the complete command line opions using ``autometa-markers -h``
+
+6. Binning
+----------
+
+This is the step where contigs are binned into genomes. There are two binning algorithms to chose from Density-Based Spatial Clustering of Applications with Noise (DBSCAN) and Hierarchical Density-Based Spatial Clustering of Applications with Noise (HDBSCAN). The default is DBSCAN.
+
+Autometa assesses clusters by examining both their completeness (number of expected single copy markers) and purity (number of single copy markers that are unique in the cluster).
+
+If we supply a taxonomy table, then that is also used to help with clustering. Otherwise, Autometa clusters solely on 5-mer frequency and coverage. 
+
+This step does the following:
+
+#. Find single-copy marker genes in the input contigs with HMMER
+#. Cluster contigs based on BH-tSNE coordinates (or any other embedding method that you have used), coverage and (optionally) taxonomy
+#. Accept clusters that are estimated to be over 20% complete and 90% pure based on single-copy marker genes. These are default papameteres and can be altered to suit your needs.
+#. Unclustered contigs leftover will be re-clustered until no more acceptable clusters are yielded
+
+If you include a taxonomy table in the, Autometa will attempt to further partition the data based on ascending taxonomic specificity (i.e. in the order phylum, class, order, family, genus, species) when clustering unclustered contigs from a previous attempt. We found that this is mainly useful if you have a highly complex metagenome (lots of species), or you have several related species at similar coverage level.
+
+Use the following command to run the binning:
+
+.. code-block:: bash
+
+    # Archaeal binning
+    autometa-binning <path/to/kmers_norm.tsv> \
+    <path/to/coverage_table.tsv> <path/to/archaea.markers.tsv> \
+    <path/to/archaea_binning.tsv> --embedded-kmers <path/to/embedded_kmers.tsv> \
+    --taxonomy <path/to/taxonomy.tsv> --clustering-method <dbscan or hdbscan> --domain archaea
+
+    # Bacterial binning
+    autometa-binning <path/to/kmers_norm.tsv> \
+    <path/to/coverage_table.tsv> <path/to/bacterial.markers.tsv> \
+    <path/to/bacteria_binning.tsv> --embedded-kmers <path/to/embedded_kmers.tsv> \
+    --taxonomy <path/to/taxonomy.tsv> --clustering-method <dbscan or hdbscan> --domain bacteria
```

Comment:
I initially couldn't run this (see Issue #133)

Reply:
Evan commented on Slack and thinks the command is wrong:

```
Looks like you’ve provided the kmers table where you should have provided the markers table. The sacc column is from the markers table

The kmer embedding is loaded already, so if you’ve copied and pasted from the docs, then the docs has the input paths wrong. Otherwise, this is not a bug, the kmers table is loaded in and then the markers table is trying to be loaded in but fails as the incorrect file was provided.
```

Reply:
Apparently the positional arguments have to be after the optional ones.